javaparser-core-testing/src/test/java/com/github/javaparser/printer/PrettyPrinterTest.java

```diff
@@ -63,6 +65,17 @@ private Optional<ConfigurationOption> getOption(PrinterConfiguration config, Con
         return config.get(new DefaultConfigurationOption(cOption));
     }
 
+    private static final ParserConfiguration.LanguageLevel storedLanguageLevel = StaticJavaParser.getParserConfiguration().getLanguageLevel();
+    @BeforeAll
+    public static void setLanguageLevel() {
+        StaticJavaParser.getParserConfiguration().setLanguageLevel(ParserConfiguration.LanguageLevel.BLEEDING_EDGE);
+    }
+
+    @AfterAll
+    public static void resetLanguageLevel() {
+        StaticJavaParser.getParserConfiguration().setLanguageLevel(storedLanguageLevel);
+    }
+
```

Comment:
There's probably a better way to do this, but I didn't want to change any defaults outside of the scope of the tests